package.json

```diff
@@ -104,7 +104,7 @@
     "eslint-plugin-async-await": "0.0.0",
     "eslint-plugin-flowtype": "2.46.1",
     "eslint-plugin-import": "2.8.0",
-    "firefox-client": "0.3.0",
+    "@cliqz-oss/firefox-client": "0.3.1",
```

Comment:
Can you move this to `dependencies` and re-enable the lint check by removing `/* eslint-disable-line import/no-extraneous-dependencies */` ?

I know it's not your fault but I just noticed it -- I think that lint rule was skipped by mistake. It only works by chance because of how npm/yarn flattens modules.

Reply:
Can do an update, but I don't think it is a mistake. `firefox-client` is a devDependency as it is only imported for flow to figure out the types. As flow syntax gets removed during build so is the import and the `firefox-client` is not required by `web-ext` directly. Does this explanation make sense?

Reply:
@chrmod yeah, you are right, the historical reasons are definitely that:
- we started to use the flow "import types" feature to have an higher coverage of the flow type checks
- later (in a different pull request) we added the additional eslint rules that checks the imports, eslint started to complain about that import and the 'eslint-disable-line' has been used as a workaround

Anyway, I agree with Kumar, I would probably prefer an additional direct dependency (in any case the dependency is already there, even if it is not currently used directly) and avoid the eslint-disable-line workaround if possible.

Do you mind to also move the new namespaced dependencies on the top of the dependencies?
```
   "dependencies": {
    "@cliqz-oss/firefox-client": "0.3.1",
    "@cliqz-oss/node-firefox-connect": "1.2.1",
     "adbkit": "2.11.0",
    ...
```

(so that the next 'npm install --save/--save-dev is not going to have to reorder them in an unrelated change)

Thanks again for your help on quickly fixing this regression.